include/handlers.h

```diff
@@ -522,7 +526,8 @@ class ImpuRegDataTask : public HssCacheTask
   ChargingAddresses _charging_addrs;
   long _http_rc;
   std::string _provided_server_name;
-  std::string _wildcard_identity;
```

Comment:
Can you comment what these variables are? (e.g. why is it important to have both?)